mlc_config.json

```diff
@@ -0,0 +1,37 @@
+{
+  "ignorePatterns": [
+    {
+      "pattern": "^https://www.postman.com/company/careers/"
+    },
+    {
+      "pattern": "^https://github.com/asyncapi/"
+    },
+    {
+      "pattern": "^https://twitter.com/"
+    },
+    {
+      "pattern": "^https://www.linkedin.com/"
+    },
+    {
+      "pattern": "^https://calendar.google.com/"
+    },
+    {
+      "pattern": "^https://github.com/issues"
+    },
+    {
+      "pattern": "^http://localhost"
+    },
+    {
+      "pattern": "^https://docs.github.com/"
+    },
+    {
+      "pattern": "^https://www.pexels.com/"
+    }

```

Comment:
Do we really need to ignore all of them?

Reply:
yeah, not sure why these are ingored. @KhudaDad414 ?

Reply:
@magicmatatjahu @alequetzalli @derberg Yes. we need to ignore all of them since we have tons of links to these websites and by checking all of the links, they think the action is spamming and return 404(or another error code, I don't exactly remember) to all of the subsequent requests.
Some websites like LinkedIn will not allow checking links without logging in first. which is why they are ignored as well.